jobs/wf2-2-upsertTEIs.js

```diff
@@ -12,69 +12,77 @@ fn(state => {
   };
 });
 
-each('patients[*]', state => {
-  const patient = state.data;
+// Prepare DHIS2 data model for patients
+fn(state => {
```

Comment:
I wasn't expecting the use of `fn` here - but it does give you a nice clean each/upsert later on!